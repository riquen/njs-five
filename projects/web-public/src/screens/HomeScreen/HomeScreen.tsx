import { Text } from "@njs-five/design-system/components/Text";

export function HomeScreen() {
  return (
    <main>
      <Text tag="h1">Home</Text>
    </main>
  );
}
