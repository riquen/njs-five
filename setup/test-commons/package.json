{
  "name": "@njs-five/test-commons",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "files": [
    "base.ts"
  ],
  "devDependencies": {
    "@babel/core": "7.17.10",
    "@babel/preset-env": "7.17.10",
    "@babel/preset-react": "7.16.7",
    "@babel/preset-typescript": "7.16.7",
    "@testing-library/react": "13.1.1",
    "babel-jest": "28.0.3",
    "jest": "27.0.0",
    "jest-styled-components": "7.0.8",
    "ts-node": "10.7.0"
  },
  "scripts": {
    "lint": "echo 'Not implemented'",
    "test": "echo 'Not implemented'"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}