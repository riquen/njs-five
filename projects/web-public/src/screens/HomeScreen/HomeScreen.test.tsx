import { customRender } from "@njs-five/test-commons/react-testing-library";
import { HomeScreen } from "./HomeScreen";

const render = customRender();

describe("<HomeScreen />", () => {
  it("should render the page", () => {
    const { container } = render(<HomeScreen />);

    expect(container).toMatchSnapshot();
  });
});
