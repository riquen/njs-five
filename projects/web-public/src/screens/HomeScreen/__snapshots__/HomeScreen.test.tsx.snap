// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> should render the page 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <main>
    <h1
      class="c0"
    >
      Home
    </h1>
  </main>
</div>
`;
